test/test_helper.rb

```diff
@@ -2,12 +2,16 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 require 'webmock'
+require 'vcr'
 
 class ActiveSupport::TestCase
-  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
   fixtures :all
 
-  # Add more helper methods to be used by all tests here...
+  VCR.configure  do |c|
+    c.cassette_library_dir = 'test/cassettes'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.